Fix/ie11

ajbarrios opened the pull request:
Hey @jhuesos !
We have fixed several IE11 visual issues in Polymer 1 and Polymer 2. We have also made some regression tests for others browsers: **Chrome, Mozilla and Safari** and everything looks great too :)

![polymer 1 ie11](https://user-images.githubusercontent.com/20030100/26968390-27b541d6-4d02-11e7-8aa4-c0f0f425d56f.png)
![polymer 2 ie11](https://user-images.githubusercontent.com/20030100/26968395-2c6cbed4-4d02-11e7-99b6-228cfd408eb5.png)
![polymer 1 custom ie11](https://user-images.githubusercontent.com/20030100/26968399-2e1effee-4d02-11e7-9954-1ca1e524d255.png)
![polymer 2 custom ie11](https://user-images.githubusercontent.com/20030100/26968402-2f93cd50-4d02-11e7-9ba3-a78edc2c2d6e.png)


Waiting for your feedback to merge

Thank you!

ajbarrios committed:
fix(css): fix css min-height for paper-item on IE11

ajbarrios committed:
fix(js): add polyfill functions for IE11

ajbarrios committed:
fix(_nodeObserver): fix new slotted nodes on IE11

ajbarrios committed:
fix(css): fix css height auto on Polymer 2.x running on IE11 for custom template

ajbarrios committed:
fix(js): remove the comment before Polymer factory method to display the documentation page

ajbarrios committed:
chore(version): update patch version

ajbarrios committed:
style(*): recover old format

ajbarrios committed:
style(comments): update custom template style comments

jhuesos commented:
Hey, many thanks!!! I will try to take a look during this weekend.

jhuesos commented on paper-autocomplete.html:
```diff
@@ -206,6 +206,43 @@
+      };
+    }
+
+    if (!Array.prototype.find) {
```
I think we should not add this polyfill. `Array.prototype.find` does not add such an important feature since you can basically achieve the same using `Array.prototype.filter` and then selecting the first element

jhuesos replied:
In fact, I had already replace your .find() method with filter(), so this is not needed

jhuesos commented:
Btw, do you know if this issue:

https://github.com/ellipticaljs/paper-autocomplete/issues/70

is solved?

jhuesos commented:
@ajbarriosleon sorry I forgot to push a change the other day, can you resolve the conflict? Thanks

ajbarrios committed:
refactor(polyfills): remove Array.find polyfill and update code with Array.filter

ajbarrios committed:
fix(merge-conflicts): resolve merge conflicts

ajbarrios commented:
Hey @jhuesos !
I have pushed some **changes:**
- I removed the **Array.find** polyfill and I replaced the code with **Array.filter.** I agree with you, it is not really useful, fewer polyfills is better :) 
- I resolved the merge conflict and tested it without any issue.

About the #70 I can't reproduce it in this branch. In my opinion, if you agree, you could merge it and keep the tracking of all old/new issues since this moment.

Thank you!

jhuesos commented:
Yeah, let's do that. this week I am extremely busy, so let's merge and I will try to do the testing there. Thanks again
